app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
> fastly will then re-cache the page with the unyanked version

and yanked_info_checksum will end up with incorrect checksum. Does that count as race condition?

PS: This PR increases number of callbacks on deletion to 5. It feels one too many. I was going to open an issue and work on it separately. May be we can consider it here itself?